src/action.ts

```diff
@@ -281,6 +290,14 @@ class Action {
     return dir
   }
 
+  private rmdir(outputsDir: string): string {
+    core.debug(`Removing directory ${outputsDir} after action finishes.`)
+    if (fs.existsSync(outputsDir)) {
+      fs.rmdirSync(outputsDir)
```

Comment:
This is currently not making any effect as the directory is usually not empty, and fails with:
> ::warning::Error removing the directory ..., error: Error: ENOTEMPTY: directory not empty, rmdir '...'

We have to apply the options mentioned [here](https://stackoverflow.com/questions/18052762/remove-directory-which-is-not-empty) so the function removes the directory recursively even if it is not empty.